system-parachains/asset-hubs/asset-hub-kusama/src/staking/mod.rs

```diff
@@ -0,0 +1,424 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Staking related config of the Asset Hub.
+//!
+//! The large pallets have their config in a sub-module, the smaller ones are defined here.
+
+pub mod bags_thresholds;
+pub mod nom_pools;
+
+use crate::{governance::StakingAdmin, *};
+use frame_election_provider_support::{ElectionDataProvider, SequentialPhragmen};
+use frame_support::traits::tokens::imbalance::ResolveTo;
+use pallet_election_provider_multi_block::{self as multi_block, SolutionAccuracyOf};
+use pallet_staking_async::UseValidatorsMap;
+use pallet_staking_async_rc_client as rc_client;
+use scale_info::TypeInfo;
+use sp_arithmetic::FixedU128;
+use sp_runtime::{
+	transaction_validity::TransactionPriority, FixedPointNumber, SaturatedConversion,
+};
+use sp_staking::SessionIndex;
+use system_parachains_constants::async_backing::MINUTES;
+use xcm::v5::prelude::*;
+
+parameter_types! {
```

Comment:
I will open a PR against this and fix all staking related things. When is your expected delivery date for this?